keyboards/xo87/rgb/rgb.h

```diff
@@ -0,0 +1,99 @@
+/* Copyright 2021 datafx
```

Comment:
What is the purpose of this file? Nothing is used?

Reply:
I believe I still need to define the layout there.  The rest was extra and was removed